Jenkinsfile

```diff
@@ -155,6 +155,7 @@ pipeline {
             sh "git submodule update --init --depth=1 --recursive"
             sh "scons --clean && scons --no-cache -j42"
             sh "scons --clean && scons --no-cache --random -j42"
+            sh "cd selfdrive/car/tests && INTERNAL_SEG_LIST='selfdrive/car/tests/test_models_segs.txt' FILEREADER_CACHE='1' pytest -n42 --dist=loadscope ./test_models.py"
```

Comment:
filereader cache isn't doing anything here

Reply:
mounted the /tmp dir